examples/run_generic_echo.py

```diff
@@ -0,0 +1,121 @@
+from pylab import *
+from pyspecdata import *
+from numpy import *
+import SpinCore_pp
+from SpinCore_pp import prog_plen
+from SpinCore_pp.ppg import generic
+import os
```

Comment:
how is this different from the next run_generic_hahn_echo?? Pick only one of the two (the one that you are using).

Reply:
the other uses echo_acq_ms this uses acq_time_ms